### DISCUSSION TITLE
Improve discoverability of `loading` state property

### DISCUSSION THREAD

**Discussion by LLCampos** [formkit/formkit#903] [Ideas]
Right now its really hard to discover that its possible to do `#default="{ state: { loading } }"` in your `FormKit` element.

- Its not mentioned in the docs
- Its not in the [`FormKitFrameworkContextState` interface](https://github.com/formkit/formkit/blob/52614833cae55fe48ea04df922c1a32efc2d869f/packages/core/src/node.ts#L517)
- It only appears in the state object when the form starts loading

Context: [this Discord conversation](https://discord.com/channels/793529058377072650/940340536747896842/1142095420483444847).

**Comment by justin-schroeder** [comment]
It is here: https://formkit.com/api-reference/context#state, but yeah, I agree it should be in more places 👍
